Movecraft/src/main/java/net/countercraft/movecraft/async/translation/TranslationTask.java

```diff
@@ -562,6 +560,10 @@ private void captureYield(@NotNull List<MovecraftLocation> harvestedBlocks) {
             //call event
             final ItemHarvestEvent harvestEvent = new ItemHarvestEvent(craft, drops, harvestedBlock.toBukkit(craft.getWorld()));
             Bukkit.getServer().getPluginManager().callEvent(harvestEvent);
+
+            if (harvestEvent.isCancelled())
+                continue;
+
```

Comment:
This will not cancel the movement and harvesting but cancel the drop of the item instead.